src/networks.js

```diff
@@ -17,8 +17,8 @@ const networks = {
   xdai: {
     chainId: 100,
     ensRegistry: '0xaafca6b0c89521752e559650206d7c925fd0e530',
-    name: 'xdai',
-    type: 'private',
+    name: 'xDai',
```

Comment:
i thin that greef suggested to rename it to xDAI

Reply:
Ah, it said that in the PR title but it's actually `xDai` https://github.com/1Hive/honey-pot/pull/19